client/ayon_core/hosts/houdini/plugins/create/create_mantra_ifd.py

```diff
@@ -25,10 +26,11 @@ def create(self, product_name, instance_data, pre_create_data):
             pre_create_data)  # type: CreatedInstance
 
         instance_node = hou.node(instance.get("instance_node"))
+        
+        filepath = self.staging_dir.format(
+            product={"name": product_name}
+        )
```

Comment:
I'd remain consistent across all creators and always format using `{ext}`. That way you can specify the exact same template in settings for all creators if a studio wants to. Otherwise they need to know specifically which needs a hardcoded ext and which doesn't.